crates/common_utils/src/events.rs

```diff
@@ -130,6 +130,7 @@ impl_api_event_type!(
         (&String, &String),
         (Option<i64>, Option<i64>, String),
         (Option<i64>, Option<i64>, id_type::MerchantId),
+        (Option<i64>, Option<i64>),
```

Comment:
Once the other change is made, you can remove this line. 
And implement the `Miscellaneous` API event type for `ListApiKeyConstraints` instead.